src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -638,24 +620,36 @@ const EnvironmentPicker = ({
                                         AzureResourceTypes.StorageBlobContainer
                                     }
                                     requiredAccessRoles={{
-                                        enforcedRoleIds: [
-                                            AzureAccessPermissionRoles['Reader']
-                                        ],
+                                        enforcedRoleIds: [],
                                         interchangeableRoleIds: [
-                                            AzureAccessPermissionRoles[
-                                                'Storage Blob Data Owner'
+                                            [
```

Comment:
Could you move the arrays out of the props and create constants or refs so the component doesn't create the array on every render?